BlueShift-iOS-SDK/BlueShiftAppData.m

```diff
@@ -51,6 +51,11 @@ - (BOOL)enablePush {
 - (void)setEnablePush:(BOOL)enablePush {
     NSString *val = enablePush ? @"YES" : @"NO";
     [[NSUserDefaults standardUserDefaults] setObject:val forKey:kBlueshiftEnablePush];
+    if (enablePush == YES) {
+        [BlueshiftLog logInfo: @"EnablePush has been manually set to YES. If app push notification permission is accepted then app will start receiving push notifications from Blueshift." withDetails:nil methodName:nil];
+    } else {
+        [BlueshiftLog logInfo: @"EnablePush has been manually set to NO. The app will stop receiveing push notifications from Blueshift. To enable receiving push notifications, set enablePush to true and fire identify call from Blueshift SDK." withDetails:nil methodName:nil];
+    }
```

Comment:
usually, this method will be called when someone sign-in to the app. so after sign in, this message won't be displayed. to take advantage of this for debugging, can we move this log to the SDK initialization code? so that on each app restart we will be able to see this log.

Reply:
Done